selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/workspaces/AddOrImportProjectFormTest.java

```diff
@@ -428,25 +428,18 @@ public void checkCreatingProject() throws Exception {
     addOrImportForm.clickOnAddButton();
     addOrImportForm.waitProjectTabAppearance(SPRING_SAMPLE_NAME);
 
-    // check closing of "Workspace Is Created" dialog window
```

Comment:
It's interesting to know why this verification had been removed?

Reply:
The workspace creation flow is changing - see the issue description and comments.